flair/data.py

```diff
@@ -461,6 +466,309 @@ def score(self):
         return self.labels[0].score
 
 
+class Tokenizer(ABC):
```

Comment:
This blows up the `data.py` file. Can you move all tokenizer related stuff to a new module `tokenizers.py`?

Reply:
Great suggestion, I moved the tokenizer implementations to `tokenization.py`. However, the abstact base class `Tokenizer` has to remain in `data.py`, since otherwise there are cross-reference issues between the two modules. This is, `Tokenizer` needs access to `Token` (from `data.py`) and for `Sentence` (from `data.py`) `Tokenizer` has to be known.